jira/client.py

```diff
@@ -909,8 +909,26 @@ def transitions(self, issue, id=None, expand=None):
             params['expand'] = expand
         return self._get_json('issue/' + str(issue) + '/transitions', params=params)['transitions']
 
+    def find_transitionid_by_name(self, issue, transition_name):
+        """
+        Get a transitionid available on the specified issue to the current user.
+        Look at https://developer.atlassian.com/static/rest/jira/6.1.html#d2e1074 for json reference
+        
+        :param issue: ID or key of the issue to get the transitions from
+        :param trans_name: iname of transition we are looking for
+        """
+        transitions_json = this.transitions(issue)
+        id = None
+
+        for transition in transtitions_json["transtions"]:
```

Comment:
@fabio-silva-movile - FYI: you have a few syntax errors / mispellings of variable names in your request, such as here on this line.